designs/developer/4.6/HPA/index.md

```diff
@@ -0,0 +1,84 @@
+---
+parent: Developer
+version: 4.6
+---
+
+# Horizontal Pod Autoscaler
+
+## Description
+
+Horizontal pod autoscalers can be applied to workloads so that pods will scale up and down as needed so resources are not being wasting when waiting for peak resources. Horizontal pod autoscalers can only be applied to Deployment/Deployment Config workloads, not Knative services. A user can add, edit, and remove a horizontal pod autoscaler. 
```

Comment:
```suggestion
Horizontal pod autoscalers can be applied to workloads so that pods will scale up and down as needed so resources are not being wasting when waiting for peak resources.
```
Just a nit really but this last sentence reads a bit strange - should it be more like waiting for peak resource requirements or time maybe?

Reply:
I can update the wording here so it sounds better.